### ISSUE TITLE
HappyBERT throws an error while running the Q&A example

### ISSUE THREAD

**Issue by KiranAkadas** [EricFillion/happy-transformer#144]
**Python 3.8.5**

```
happy_bert = HappyBERT()
question = "Who does Ernie live with?"
text = "Ernie is an orange Muppet character on the long running PBS and HBO children's television show Sesame Street. He and his roommate Bert form the comic duo Bert and Ernie, one of the program's centerpieces, with Ernie acting the role of the naïve troublemaker and Bert the world weary foil."  # Source: https://en.wikipedia.org/wiki/Ernie_(Sesame_Street)
result = happy_bert.answer_question(question,text)
print(type(result)) # prints: <class 'str'>
print(result) # prints: bert
```

The example given in the official documentation gives the following error when run:

```
TypeError                                 Traceback (most recent call last)
<ipython-input-8-618d260e51a7> in <module>()
      2 question = "Who does Ernie live with?"
      3 text = "Ernie is an orange Muppet character on the long running PBS and HBO children's television show Sesame Street. He and his roommate Bert form the comic duo Bert and Ernie, one of the program's centerpieces, with Ernie acting the role of the naïve troublemaker and Bert the world weary foil."  # Source: https://en.wikipedia.org/wiki/Ernie_(Sesame_Street)
----> 4 result = happy_bert.answer_question(question,text)
      5 print(type(result)) # prints: <class 'str'>
      6 print(result) # prints: bert

/usr/local/lib/python3.6/dist-packages/happytransformer/happy_bert.py in answer_question(self, question, text)
    140                                                token_type_ids=segment_tensor)
    141         all_tokens = self.tokenizer.convert_ids_to_tokens(input_ids)
--> 142         answer_list = all_tokens[torch.argmax(start_scores):
    143                                  torch.argmax(end_scores)+1]
    144         answer = self.tokenizer.convert_tokens_to_string(answer_list)

TypeError: argmax(): argument 'input' (position 1) must be Tensor, not str
```

**Comment by Murtaza-arif** [comment]
@KiranAkadas Can you provide some more details?
Because it is working fine for me 
I am using python 3.9

**Comment by ted537** [comment]
You say Python  3.8.5 but your path is at `/usr/local/lib/python3.6`

This is probably related to the dependencies in `setup.py`,  as they are not versioned currently

**Comment by KiranAkadas** [comment]
Yea, Missed it. As you mentioned there was a dependency mismatch. Solved it by using a virtual env

**Comment by skymansandy** [comment]
Hey @ted537, @EricFillion. I faced this error too. After some digging, found this [SO post](https://stackoverflow.com/questions/65132144/bertmodel-transformers-outputs-string-instead-of-tensor) and changed accordingly (i.e., changed `start_scores`, `end_scores` to `scores` array and used `scores[0]`, `scores[1]`) That worked.

So I had to change the library code this way to make it work. thoughts?

PS: I'm on python 3.8.5.
